Tribler/Test/test_gui_dialogs.py

```diff
@@ -176,18 +175,26 @@ def disable_dispersy():
         self.startTest(disable_dispersy)
 
     def test_confirmationdialog(self):
```

Comment:
Go back to the original version of the test, and understand why it gets stuck. Rewriting stuff in the hopes that it will get fixed is not the way to go.